client/client-multi/client-c/src/kaa/avro_src/io.c

```diff
@@ -117,13 +118,13 @@ int avro_write(avro_writer_t writer, void *buf, int64_t len)
 void avro_reader_free(avro_reader_t reader)
 {
 	if (reader) {
-		free(reader);
+		KAA_FREE(reader);
 	}
 }
 
 void avro_writer_free(avro_writer_t writer)
 {
 	if (writer) {
-		free(writer);
+	    KAA_FREE(writer);
```

Comment:
I know it isn't your fault.  But, please, replace tab symbols with 4 spaces all around the file.